{"url": "https://api.github.com/repos/datalad/datalad/issues/3085", "repository_url": "https://api.github.com/repos/datalad/datalad", "labels_url": "https://api.github.com/repos/datalad/datalad/issues/3085/labels{/name}", "comments_url": "https://api.github.com/repos/datalad/datalad/issues/3085/comments", "events_url": "https://api.github.com/repos/datalad/datalad/issues/3085/events", "html_url": "https://github.com/datalad/datalad/issues/3085", "id": 389382077, "node_id": "MDU6SXNzdWUzODkzODIwNzc=", "number": 3085, "title": "datalad SingularityHub container does not run on newer versions of singularity", "user": {"login": "shots47s", "id": 9719, "node_id": "MDQ6VXNlcjk3MTk=", "avatar_url": "https://avatars.githubusercontent.com/u/9719?v=4", "gravatar_id": "", "url": "https://api.github.com/users/shots47s", "html_url": "https://github.com/shots47s", "followers_url": "https://api.github.com/users/shots47s/followers", "following_url": "https://api.github.com/users/shots47s/following{/other_user}", "gists_url": "https://api.github.com/users/shots47s/gists{/gist_id}", "starred_url": "https://api.github.com/users/shots47s/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/shots47s/subscriptions", "organizations_url": "https://api.github.com/users/shots47s/orgs", "repos_url": "https://api.github.com/users/shots47s/repos", "events_url": "https://api.github.com/users/shots47s/events{/privacy}", "received_events_url": "https://api.github.com/users/shots47s/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "closed", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 3, "created_at": "2018-12-10T16:30:29Z", "updated_at": "2018-12-11T13:36:02Z", "closed_at": "2018-12-11T00:33:12Z", "author_association": "NONE", "active_lock_reason": null, "body": "#### What is the problem?\r\n\r\nThe instructions on the datalad website give the very nice option of running datalad with a simple singularity pull.  However if the machine that you are launching on has singularity > 2.4.1, then you receive errors.\r\n\r\n#### What steps will reproduce the problem?\r\nOn a machine that has singularity 2.5.1-dist:\r\n`singularity pull shub://datalad/datalad:fullmaster`\r\nthen\r\n`datalad install -r -g -s ///abide/RawDataBIDS/CMU_a ./X`\r\n\r\n#### What version of DataLad are you using (run `datalad --version`)? On what operating system (consider running `datalad wtf`)?\r\n\r\n```\r\nDataLad\r\n=======\r\nVersion     : 0.10.2\r\nFull version: 0.10.2\r\n\r\nSystem\r\n======\r\nOS          : posix Linux 4.4.24-1.el7.elrepo.x86_64 #1 SMP Fri Oct 7 10:41:16 EDT 2016\r\nDistribution: debian/9.4\r\n\r\nLocale/Encoding\r\n===============\r\ndefault        : utf-8\r\nfilesystem     : ascii\r\nlocale.prefered: ANSI_X3.4-1968\r\n\r\nEnvironment\r\n===========\r\nPATH='/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'\r\nLANG='en_CA.UTF-8'\r\nGIT_PYTHON_GIT_EXECUTABLE='/usr/lib/git-annex.linux/git'\r\n\r\nExternals\r\n=========\r\nappdirs=1.4.3\r\nboto=2.49.0\r\ncmd:annex=6.20180626+gitg12cd64369-1~ndall+1\r\ncmd:git=2.18.0\r\ncmd:system-git=2.11.0\r\ncmd:system-ssh=7.4p1\r\nexifread=2.1.2\r\ngit=2.1.10\r\ngitdb=2.0.3\r\nhumanize=0.5.1\r\niso8601=0.1.12\r\nmsgpack=0.5.6\r\nmutagen=1.40.0\r\nrequests=2.19.1\r\nsix=1.10.0\r\ntqdm=4.23.4\r\nwrapt=1.10.11\r\n\r\nInstalled extensions\r\n====================\r\n\r\n\r\nKnown metadata extractors\r\n=========================\r\ndatalad_core: OK\r\nimage: OK\r\ndatalad_rfc822: OK\r\nfrictionless_datapackage: OK\r\ndatacite: OK\r\nannex: OK\r\nexif: OK\r\naudio: OK\r\nxmp: OK\r\n```\r\n#### Is there anything else that would be useful to know in this context?\r\n\r\nDon't think so.\r\n\r\n#### Have you had any success using DataLad before? (to assess your expertise/prior luck.  We would welcome your testimonial additions to \r\nhttps://github.com/datalad/datalad/wiki/Testimonials as well)\r\n\r\nThis has worked on machines that have singularity-2.4.1-dist without issue.\r\n", "closed_by": {"login": "yarikoptic", "id": 39889, "node_id": "MDQ6VXNlcjM5ODg5", "avatar_url": "https://avatars.githubusercontent.com/u/39889?v=4", "gravatar_id": "", "url": "https://api.github.com/users/yarikoptic", "html_url": "https://github.com/yarikoptic", "followers_url": "https://api.github.com/users/yarikoptic/followers", "following_url": "https://api.github.com/users/yarikoptic/following{/other_user}", "gists_url": "https://api.github.com/users/yarikoptic/gists{/gist_id}", "starred_url": "https://api.github.com/users/yarikoptic/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/yarikoptic/subscriptions", "organizations_url": "https://api.github.com/users/yarikoptic/orgs", "repos_url": "https://api.github.com/users/yarikoptic/repos", "events_url": "https://api.github.com/users/yarikoptic/events{/privacy}", "received_events_url": "https://api.github.com/users/yarikoptic/received_events", "type": "User", "site_admin": false}, "reactions": {"url": "https://api.github.com/repos/datalad/datalad/issues/3085/reactions", "total_count": 0, "+1": 0, "-1": 0, "laugh": 0, "hooray": 0, "confused": 0, "heart": 0, "rocket": 0, "eyes": 0}, "timeline_url": "https://api.github.com/repos/datalad/datalad/issues/3085/timeline", "performed_via_github_app": null}